lmms_eval/tasks/gmai/utils.py

```
@@ -0,0 +1,110 @@
+import io
+import base64
+import logging
+import re
+import random 
```

Ruff (0.12.2)

`random` imported but unused

Remove unused import: `random`

(F401)